packages/cadl-rlc-test/test/parametrizedHost/cadl-output/review/parametrized-host.api.md

```diff
@@ -57,11 +58,18 @@ export interface ListCollections200Response extends HttpResponse {
     status: "200";
 }
 
+// @public (undocumented)
+export interface ListCollectionsDefaultHeaders {
+    "x-ms-error-code"?: string;
```

Comment:
The header is back? I remember we have it previous

Reply:
no idea, I didn't change the cadl file about header.